azure/src/test/java/sunstone/azure/armTemplates/di/AzureWebAppTests.java

```diff
@@ -21,27 +21,25 @@
 import static org.assertj.core.api.Assertions.assertThat;
 
 @WithAzureArmTemplate(template = "sunstone/azure/armTemplates/eapWebApp.json",
-        parameters = {@Parameter(k = "appName", v = AzureTestConstants.instanceName)}, group = AzureWebAppTests.group)
```

Comment:
Can we have unique test class specific group? Like in `MultipleAzArmTemplatesRepetableTest`.